src/reference/user-surveys.md

```diff
@@ -0,0 +1,35 @@
+---
+title: User surveys
+description: Why users see a survey announcement, how the data is used, and how to disable.
+---
+
+## Why do I see a survey announcement?
+If you have not opted-out of Flutter’s 
+[analytics and crash reporting]({{site.url}}/reference/crash-reporting), 
+you may receive a survey announcement in your IDE. 
+
+We run two types of surveys:
+1. Quarterly surveys - For these surveys, we notify all active
+ Flutter and Dart users to optionally take the survey.
+2. Ad-hoc surveys (experimental): We design and deploy these 
+surveys when we want to learn more about specific topics. 
+You may see a survey announcement if you meet the criteria of 
+the survey  based on data stored in the 
+`dart-flutter-telemetry.log` file on your disk.
```

Comment:
Same thing here for where they can find this file in the HOME directory on macOS/linux and on windows it will be in the `AppData` directory

Reply:
Thanks!! Does the following description make sense? 

Additionally, you may review the recently reported data in the
`.dart-tool/dart-flutter-telemetry.log` file under the home directory
`~/` (macOS/Linux) or the `<username>\AppData` (Windows) directory.

Reply:
Perfectly put!